integrated_channels/sap_success_factors/exporters/content_metadata.py

```diff
@@ -129,6 +130,22 @@ def transform_schedule(self, content_metadata_item):  # pylint: disable=unused-a
         """
         return []
 
+    def transform_price(self, content_metadata_item):
+        """
+        Return the current course run's price.
+        """
+        current_course_run = None
+        if self.enterprise_configuration.show_course_price:
+            for course_run in content_metadata_item.get('course_runs', []):
+                if course_run['availability'] == 'Current':
+                    current_course_run = course_run
+        return [
+            {
+                "currency": "USD",
+                "value": current_course_run['first_enrollable_paid_seat_price'] if current_course_run else 0.0
```

Comment:
can we extra protection here in case we don't have `first_enrollable_paid_seat_price` key in `current_course_run`
```
current_course_run['first_enrollable_paid_seat_price'] if current_course_run and current_course_run.get('first_enrollable_paid_seat_price') else 0.0
```